lib/versions-complete.xml

```diff
@@ -8981,7 +8981,7 @@
 			</library>
 			<library>
 				<file-name>lib/development/velocity.jar</file-name>
-				<version>1.6.4</version>
+				<version>1.6.4.LIFERAY-PATCHED-2</version>
```

Comment:
@tinatian I forget whether it was versions.html or versions-complete.xml, but one of those files wasn't getting updated when I ran `ant build-lib-versions`.

Also, I had a lot of other problems with the `ant build-lib-versions` command recently. It would make a lot of extra changes that weren't relevant. For example, it would make timezone-related changes where it modify all the the `current-version-publish-date` and `latest-version-publish-date` to be a few hours later than what they are now. And sometimes I think it would change the license metadata for some libraries as wel.